server/src/test/resources/version.properties

```diff
@@ -1,2 +0,0 @@
-version=${version}
-release=${release}
```

Comment:
The contents of this were deleted, but the file was left? Is the file needed at all? what was it's original purpose anyway?

Reply:
Both files (& its content) are still present in new structure. 
We refer these files within source code for checking compatibility.